clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -573,18 +573,6 @@ private void maybeOverrideClientId(Map<String, Object> configs) {
         return newConfigs;
     }
 
-    public static Properties addDeserializerToConfig(Properties properties,
```

Comment:
> did you get around to update the PR with a variant that removes the no-longer-used addDeserializerToConfig() method?

done !

Reply:
We cannot remove this method since it's in a class that is part of Kafka's public API. We should probably deprecate it, but we need a KIP for that.